package-lock.json

```diff
@@ -1,8 +1,8832 @@
 {
   "name": "aria-practices",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
NPM 7 re-writes the entire file with the new lockfile version

Reply:
Is this good or bad?

Reply:
Not sure, there were some incompatibilities with Dependabot at one time, but I'm not sure if that's still the case